Helper/Requests.php

```diff
@@ -127,18 +125,39 @@ public function buildCustomerData(
                 if ($customerEmail = $billingAddress->getEmail()) {
                     $request['shopperEmail'] = $customerEmail;
                 }
-
                 if ($customerTelephone = trim($billingAddress->getTelephone())) {
                     $request['telephoneNumber'] = $customerTelephone;
                 }
-
                 if ($firstName = $billingAddress->getFirstname()) {
                     $request['shopperName']['firstName'] = $firstName;
                 }
 
                 if ($lastName = $billingAddress->getLastname()) {
                     $request['shopperName']['lastName'] = $lastName;
                 }
+
+                if ($dateOfBirth = $billingAddress->getDateOfBirth()) {
+                    $request['paymentMethod']['personalDetails']['dateOfBirth'] = $dateOfBirth;
+                }
+            }
+
+            //if payment method is afterpay_default override the parameters with the provided ones
+            if ($paymentMethod == 'afterpay_default') {
```

Comment:
should be done for not just afterpay_default, if these values are sent from the frontend we can use them instead of the billing address values

Reply:
I am gonna integrate the if statements with the existing ones